### PR TITLE
fix: SSR missmatch

### LINKED ISSUE #1813: [BUG] SSR missmatch for SfHeaderNavigationItem and SfBanner on Mobile
**Describe the bug**
Components using mapMobileObserver are triggering SSR missmatch when refreshing in mobile view. isMobile is returning false on the server and true on the client.

The header bar doesn't render correctly and sticks to the top of the screen.

**How to reproduce**
1. Go to Chrome mobile view
2. Reload page
3. See error in console

**Expected behavior**
SSR and Client match

**Actual behavior**
Missmatch of isMobile causing errors. Header bar not rendering correctly.

**Code examples**

```
//Write code here
```
```
<template>
  <SfHeader
    logo="https://www.svgrepo.com/show/9395/ribbon-banner-silhouette.svg"
    :title="title"
    :active-icon="activeIcon"
    :search-placeholder="searchPlaceholder"
    :search-value="searchValue"
    :cart-icon="cartIcon"
    :wishlist-icon="wishlistIcon"
    :is-sticky="isSticky"
    :account-icon="accountIcon"
    :cart-items-qty="cartItemsQty"
    :wishlist-items-qty="wishlistItemsQty"
    @click:cart="alert('@click:cart')"
    @click:wishlist="alert('@click:wishlist')"
    @click:account="alert('@click:account')"
    @change:search="searchValue = $event"
  >
    <template #navigation>
      <SfHeaderNavigationItem
        v-for="item in navigation"
        :key="item.title"
        :label="item.title"
        :link="item.link"
      />
    </template>
  </SfHeader>
</template>

<script>
import { SfHeader, SfLink } from "@storefront-ui/vue";

export default {
  name: "Header",
  components: {
    SfHeader,
    SfLink,
  },
  data() {
    return {
      navigation: [
        {title: "Cat1", link: "/cat1"},
        {title: "Cat2", link: "/cat2"},
        {title: "Cat3", link: "/cat3"}
      ],
      searchValue: "",
      title: "",
      activeIcon: "account",
      isSticky: false,
      searchPlaceholder: "Search for items",
      cartIcon: "empty_cart",
      wishlistIcon: "heart",
      accountIcon: "profile",
      cartItemsQty: "0",
      wishlistItemsQty: "0",
    };
  }
};
</script>
```

**Screenshots**
After adding `console.log('isMobile: ' + this.isMobile)` in SfHeaderNavigationItem isVisibleOnMobile computed.
image

**Additional context**
Node: 12.22.1
Nuxt: 2.15.3
Storefront UI: 0.10.6

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes #1813

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Review by Baroshem — APPROVED** [review]
Great work, console tab clean as banking account after Friday night

File: packages/vue/src/stories/releases/v0.10.7/v0.10.7.stories.mdx [hunk 1]
```diff
@@ -23,3 +23,5 @@ import { Meta } from "@storybook/addon-docs/blocks";
 ## 🐛 Fixes
 
 - SfAccordion - header text should be aligned left when multiline
+- SfBanner, SfCategoryCard, SfTile, SfHeader, SfHero: fixed SSR missmatch 
+- SfAccordion - header text should be aligned left when multiline
```
Comment (Baroshem) [inline packages/vue/src/stories/releases/v0.10.7/v0.10.7.stories.mdx:27]: You have the same fix note two times ;)

**Review by Baroshem — COMMENTED** [review]
One small thing @justyna-13

**Review by AdamPawlinski — COMMENTED** [review]
@justyna-13 Could you please check SfHero, because when I tested it on nuxt project mismatch errors appears  But I'm not sure if it is sth that happens only on my local machine or real issue. 
image

**Comment by AdamPawlinski** [comment]
I tested it on another nuxt project and it works fine so probably there were some other issues causing this behaviour. Sorry to disrupt you. This is perfect work :+1:

**Review by AdamPawlinski — APPROVED** [review]

**Pull request merged**